frontend/cypress/tests/chartView/chartView_showTags.cy.js

```diff
@@ -64,4 +64,23 @@ describe('show tags', () => {
         .should('equal', 'https://github.com/reposense/RepoSense/releases/tag/v1.0');
     });
   });
+
+  it('group by authors works with show tags', () => {
+    cy.get('div.mui-select.grouping > select:visible')
+      .select('groupByAuthors');
+
+    cy.get('div.mui-select.sort-within-group > select:visible')
+      .select('title dsc');
+
+    cy.get('#summary label.show-tags > input:visible')
+      .should('be.visible')
+      .check()
+      .should('be.checked');
+
+    cy.get('.summary-chart:first summary-chart__title--tags')
+      .should('not.exist');
+
+    cy.get('.summary-chart:nth-child(2) summary-chart__title--tags')
+      .should('not.exist');
```

Comment:
Based on what I see in the testing data, shouldn't the second chart have tags? This seems to check if they don't exist instead.

Reply:
fixed!